pkgs/games/BeatSaberModManager/default.nix

```diff
@@ -47,6 +49,12 @@ buildDotnetModule rec {
     fontconfig
   ];
 
+  # Required for OneClick
+  postFixup = ''
```

Comment:
You can just set makeWrapperArgs, see pkgs/build-support/dotnet/build-dotnet-module/hooks/dotnet-fixup-hook.sh

Reply:
This produces a much larger wrapper which also injects the dotnet runtime and runtimeDeps. That doesn't sound bad but I'm not 100% sure I'd want that.

Reply:
Currently you're wrapping that "large wrapper", it's necessary to make dotnet apps work

Reply:
I see, indeed there's already a wrapper. Thanks for the insight!